portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,32 @@
+package main.java.com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+        String paramName = request.getParameter("name");
+ 
+        String paramMail = request.getParameter("email");
+
+        String paramMessage = request.getParameter("message");
+
+        String paramNum = request.getParameter("number");
+ 
+        System.out.println(paramName + " submitted: '" + paramMail + "' as email, " + paramNum + " as number, and '" + paramMessage + "' as a message");
```

Comment:
this is debugging code right? If so, please delete before submitting/merging final product

Reply:
No, this line of code notifies the Google Cloud log that a POST request just got in, and prints out the message in the log system

Reply:
This line of code notifies the Google Clouds logging system that POST request has been made and prints the sender's info and message in the log system